### DISCUSSION TITLE
Logitech webcam

### DISCUSSION THREAD

**Discussion by ManalKR** [bonsai-rx/bonsai#1295] [Q&A]
Hi,

I want to use logitech webcam. For this I am using CameraCapture node, but bonsai keep recognizing the computer's built in webcam. So, how do I make it recognize my logitech webcam instead?

Thanks!

**Comment by bruno-f-cruz** [comment]
Change the `Index` property.
https://bonsai-rx.org/docs/api/Bonsai.Vision.CameraCapture.html#Bonsai_Vision_CameraCapture_Index

**Reply by vlkuzun** [reply]
Hi Bruno
Just to add to this question if my camera were to be attached through a USB port, what value would be entered into the Index property?

**Reply by bruno-f-cruz** [reply]
No idea. I would just try different indices until i find the correct one. I don't think the opencv backend supports any other sort of indexing tbh
